### ISSUE TITLE
[FR] Button to sync latest version of all pages from cloud

### ISSUE THREAD

**Issue by emmggi** [AppFlowy-IO/AppFlowy#5345]
### Description

I need a button to retrieve the latest version of all pages from the cloud. If we make changes to a big number of pages we need to open them all on other devices, one by one, and wait to sync. At the moment, sync also takes some time so it's even slower. Working on the newest version of the page will also reduce conflicts and prevent unexpected behavior.

### Impact

All users, offline users even more.

**Comment by diegovskytl** [comment]
Yup, totally necessary feature.
Also, a complete update from the cloud should run at startup for all versions in order to ease sync process.

**Comment by emmggi** [comment]
> Yup, totally necessary feature. Also, a complete update from the cloud should run at startup for all versions in order to ease sync process.

Sounds even better.

**Comment by unknoxx** [comment]
I agree, at the moment the sync method isn't productive at all your idea is golden
